include/libopencm3/stm32/f7/fmc.h

```diff
@@ -0,0 +1,256 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2013 Chuck McManis <cmcmanis@mcmanis.com>
```

Comment:
This entire file appears to be a duplicate of the f4 file?  I couldn't spot any differences, please make it a common file too.

Reply:
the addresses differ, see the f4-to-f7 migration application note an4660, chapter 2.5 (http://www.st.com/content/ccc/resource/technical/document/application_note/73/76/21/47/ef/d0/4c/16/DM00164538.pdf/files/DM00164538.pdf/jcr:content/translations/en.DM00164538.pdf)

Reply:
but apparently i messed this up somehow